state/store.go

```diff
@@ -66,6 +70,8 @@ type Store interface {
 	LoadLastABCIResponse(int64) (*cmtstate.ABCIResponses, error)
 	// LoadConsensusParams loads the consensus params for a given height
 	LoadConsensusParams(int64) (cmtproto.ConsensusParams, error)
+	// LoadConsensusTimeoutsInfo loads timeouts info for a given height
```

Comment:
[nit]

```suggestion
	// LoadConsensusTimeoutsInfo loads timeout info for a given height
```

Reply:
The term 'timeouts' refers to multiple timeout values. Could you explain why you're suggesting the singular form here?

Reply:
I noticed this as well. I think it's because `info` somewhat implies plurality

Reply:
resolving as we removed this method entirely